packages/reactstrap-validation-select/src/AvSelectField.js

```diff
@@ -41,9 +41,12 @@ class AvSelectField extends Component {
 
     const validation = this.context.FormCtrl ? this.context.FormCtrl.getInputState(name) : {};
     let feedback = null;
+    let ariaFeedback = {};
     if (validation.errorMessage) {
       const feedbackClasses = classNames('d-block', feedbackClass);
-      feedback = <AvFeedback className={feedbackClasses}>{validation.errorMessage}</AvFeedback>;
+      const feedbackId = `${id}-feedback`;
+      feedback = <AvFeedback className={feedbackClasses} id={feedbackId}>{validation.errorMessage}</AvFeedback>;
+      ariaFeedback = {feedbackId: feedbackId, errorMessage: validation.errorMessage}
```

Comment:
looks like linter is upset about not using property shorthand for feedbackId. might as well fix it so it doesn't pop up in future PRs.